config/.jshintrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Does this disable jshint on config or set it to the default? Wouldn't it be better to add it to the ignored dirs in the grunt config? Don't we actually _want_ jshint on config?